src/Json.Schema.ToDotNet.UnitTests/Assert.cs

```diff
@@ -10,10 +10,13 @@ internal static class Assert
     {
         internal static void FileContentsMatchExpectedContents(
             TestFileSystem testFileSystem,
-            IDictionary<string, ExpectedContents> expectedContentsDictionary)
+            IDictionary<string, ExpectedContents> expectedContentsDictionary,
+            bool generateEqualityComparers)
```

Comment:
>generateEqualityComparers [](start = 17, length = 25)

Before this, every unit test generated an equality comparer in addition to the class, and this assertion method relied on that. The test I just added has no implications for the equality comparer, so I wanted to make it possible to write a test that just generated the class. All the other changes, except for the one new test and the actual implementation, flow from this change.